doc/flame/components.md

```diff
@@ -788,6 +789,18 @@ This is an example of how a quarter-length map looks like:
 
 ![An example of a isometric map with selector](../images/isometric.png)
 
+The component supports negative indexing starting from -2 *(as -1 is reserved for empty tile)* 
+which returns the sprite from the bottom left to the bottom right using negative indexing and
+wraps to the upper rows on overflow of the tileset's image width. 
+
+Use the `propSize` property to give a custom size to negative indexed elements 
+*(This can be used to add props and art such as tiles which rise higher than most tiles)*.
+
+The following is an example of rendering props as tiles. In this example it is specifically the
+mirrors, some of which are higher than the normal tiles.
+
+![An example use case of props](../images/isometricPropsThumb.png)
```

Comment:
It would maybe be better if we had an example for it and reference that image here so people can also see the code behind the visualization

Reply:
Yeah, but the source code for the image larger than what a typical example would be. I could add a link to the project which currently wouldn't work but will work as I will set the project public soon anyways. Should I do that or make a smaller example in the examples for flame?

Reply:
How long is the wxanple currently? We have considered putting more full-blown examples into the example page anyways.

Reply:
The current game implementation of the levelLoader page as we are seeing here is about 250-300 lines but that includes things like camera movement, tap, state management etc. not just rendering. 

I am thinking of adding just a simple example for the flame examples, while we are at it.